modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -201,17 +201,17 @@ $(document).ready(function() {
         let $target = $(event.currentTarget);
         let tabName = '';
 
-        if ($target.hasClass('master-tabs')) {
+        if ($($target).closest('.master-tabs').length) {
```

Comment:
@ayumihamsaki My turn to jump in now (apologies :P) - the `$target` variable is already a jQuery object by this point, so you should not need the outer `$(...)` jQuery constructor call. Would you mind changing all references of `$($target)` to just `$target`?

Reply:
@bennothommo was quicker this time, sorry @ayumihamsaki 😄 Haven't yet waked up)

Reply:
haha sure will do, thanks